src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,8 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
Don't use * selector.

Reply:
We can use selector * to set box-sizing but we cant reset other styles in this selector.
So I used it for set-only box-sizing, it allowed.

Reply:
> 
We can use selector * to set box-sizing but we cant reset other styles in this selector.
So I used it for set-only box-sizing, it allowed.

Reply:
> Don't use * selector.

We can use selector * to set box-sizing but we cant reset other styles in this selector.
So I used it for set-only box-sizing, it allowed.

Reply:
Agree, `box-sizing` is an exception